sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -5646,7 +5658,7 @@ pi_result piEnqueueMemBufferRead(pi_queue Queue, pi_mem Src,
   PI_CALL(Src->getZeHandle(ZeHandleSrc, _pi_mem::read_only, Queue->Device));
   return enqueueMemCopyHelper(PI_COMMAND_TYPE_MEM_BUFFER_READ, Queue, Dst,
                               BlockingRead, Size, ZeHandleSrc + Offset,
-                              NumEventsInWaitList, EventWaitList, Event);
+                              NumEventsInWaitList, EventWaitList, Event, true);
```

Comment:
```suggestion
                              NumEventsInWaitList, EventWaitList, Event, /* PreferCopyEngine */ true);
```
and clang-format.

Alternative solution: use named variable instead of "magic constant".

Reply:
OK